### PR TITLE
Adding pfSense firewall rules.

### PR DESCRIPTION
**Pull request by mjflve**
Hi,

This PR adds some rules fir pfSense firewall block events. Please review, thanks!

Regards,

### CODE REVIEW

File: rules/0540-pfsense_rules.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  Created by Wazuh, Inc. <support@wazuh.com>.
+  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  ID: 87700--87799
+-->
+
+<group name="pfsense,">
+  <rule id="87700" level="0">
+    <decoded_as>pf</decoded_as>
+    <program_name>filterlog</program_name>
+    <description>pFsense firewall rules grouped.</description>
```
Comment (128432b4117) [inline rules/0540-pfsense_rules.xml]: Spelling is "pfSense" ?

File: rules/0540-pfsense_rules.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  </rule>
+
+  <!-- We don't log firewall events, because they go
+    -  to their own log file.
+    -->
+  <rule id="87701" level="5">
+    <if_sid>87700</if_sid>
+    <action>block</action>
+    <options>no_log</options>
+    <description>pFsense firewall drop event.</description>
```
Comment (128432b4117) [inline rules/0540-pfsense_rules.xml]: Spelling is "pfSense" ?

File: rules/0540-pfsense_rules.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <group>firewall_block,pci_dss_1.4,gpg13_4.12,</group>
+  </rule>
+
+  <rule id="87702" level="10" frequency="16" timeframe="45" ignore="240">
+    <if_matched_sid>87701</if_matched_sid>
+    <same_source_ip />
+    <description>Multiple pFsense firewall blocks events from same source.</description>
```
Comment (128432b4117) [inline rules/0540-pfsense_rules.xml]: Spelling is "pfSense" ?

**Comment by jesuslinares** [comment]
Great!. Thanks.

**Pull request merged**